modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -149,7 +149,12 @@ The following table lists all ports that must be open, along with the respective
 | 443 | TCP | HTTPS | Internet | Controller nodes | Allow inbound requests to Mules
 | 443 | TCP | AMQP over WebSockets | Controller nodes | Internet | Anypoint Platform management services
 | 443 | TCP | HTTPS | All nodes | Internet | API Manager policy updates, API Analytics Ingestion, and Resource retrieval (application files, container images).
+| 443 (v1.8.50, or later) | TCP | Lumberjack | All nodes | Internet | Anypoint Monitoring, Anypoint Visualizer
 | 5044 | TCP | Lumberjack | All nodes | Internet | Anypoint Monitoring, Anypoint Visualizer
```

Comment:
Can you add `deprecated` to 5044 as well? Similar to how we added to https://docs.mulesoft.com/runtime-fabric/1.8/install-prereqs#port-ip-addresses-and-hostnames-to-whitelist

Reply:
Added (deprecated)